src/main/kotlin/org/rust/lang/core/CompilerFeature.kt

```diff
@@ -21,14 +21,31 @@ import org.rust.lang.core.stubs.index.RsFeatureIndex
 import org.rust.lang.utils.RsDiagnostic
 import org.rust.lang.utils.addToHolder
 
-data class CompilerFeature(val name: String, val state: FeatureState, val since: SemVer) {
-    constructor(name: String, state: FeatureState, since: String) : this(name, state, SemVer.parseFromText(since)!!)
+class CompilerFeature(
+    val name: String,
+    val state: FeatureState,
+    val since: SemVer?,
+    cache: Boolean = false
```

Comment:
Why default value is `false`?
You have more than 400 usages with non default value and only one with default one.

Reply:
@Undin Fixed.